src/style.css

```diff
@@ -0,0 +1,25 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+  justify-content: space-between;
+  width: 96px;
```

Comment:
try to remove justify content and width rules and add margin-right 4px to stars__star block instead